cloudferry_devlab/cloudferry_devlab/tests/testcases/test_resource_migration.py

```diff
@@ -52,21 +52,33 @@ def validate_resource_parameter_in_dst(self, src_list, dst_list,
             name_attr = 'display_name'
         for i in src_list:
             for j in dst_list:
-                if getattr(i, name_attr) != getattr(j, name_attr):
+                src_attr = getattr(i, name_attr)
+                dst_attr = getattr(j, name_attr)
+                src_param = getattr(i, parameter, None)
+                dst_param = getattr(j, parameter, None)
+                if resource_name == 'tenant':
+                    src_attr = src_attr.lower()
+                    dst_attr = dst_attr.lower()
+                    if parameter == 'name':
```

Comment:
PEP8 issue resolved. I used another pep8 tool which asked to enter a blank line at the end of the file, hence I had added. Removed now to comply with this tool.